types/multicoin-address-validator/index.d.ts

```diff
@@ -3,7 +3,7 @@ export as namespace WAValidator;
 export function validate(address: string, currencyNameOrSymbol?: string, opts?: ValidateOpts | string): boolean;
 export function getCurrencies(): Currency[];
 export function findCurrency(symbol: string): Currency | null;
-
+export type NetworkType = "prod" | "testnet" | "both";
```

Comment:
It looks like at least one other value, `"stagenet"`, is possible.

https://github.com/christsim/multicoin-address-validator/blob/d5f41ba84e0b5e5f7e30a8b303561a68992b8874/src/monero_validator.js#L24